debian/control

```diff
@@ -4,11 +4,16 @@ Priority: extra
 Maintainer: Sipwise Development Team <support@sipwise.com>
 Build-Depends: debhelper (>= 5),
                iptables-dev (>= 1.4),
-               libavcodec-dev,
-               libavfilter-dev,
-               libavformat-dev,
-               libavresample-dev,
-               libavutil-dev,
+               libavcodec-dev (>= 6:10),
+               libavfilter-dev (>= 6:10),
+               libavformat-dev (>= 6:10),
+               libavresample-dev (>= 6:10),
+               libavutil-dev (>= 6:10),
+               libavcodec55 (>= 6:10) | libavcodec56, 
+               libavformat55 (>= 6:10) | libavformat56, 
+               libavfilter4 (>= 6:10) | libavfilter5, 
+               libopus0 (>= 1.1),
+               libswscale2 (>= 6:10) | libswscale3, 
```

Comment:
Just had a private chat with Richard regarding the Build-Dependencies and as he (absolutely correctly) pointed out e.g. the libavcodec* versions are moving forward fast so this might be annoying to maintain. Instead we (as usual) should have Build-Depends on their according -dev packages (like e.g. libavcodec-dev instead of libavcodec55 ...), or is there anything specific about that why the non-dev packages are needed too?